wrappers/android/app/src/main/java/com/example/zxingcppdemo/MainActivity.kt

```diff
@@ -61,14 +64,37 @@ class MainActivity : AppCompatActivity() {
 		binding.capture.setOnClickListener {
 			// Disable all camera controls
 			it.isEnabled = false
-
-			//TODO: save image
-
+			takePhoto()
 			// Re-enable camera controls
 			it.isEnabled = true
 		}
 	}
 
+	private fun takePhoto() {
+		val imageCapture = imageCapture ?: return

```

Comment:
The whole PR looks very nice. Thanks a lot! Just one question: what is this line supposed to mean? You introduce a local variable with the same name and same value as the outer scope one? Is this supposed to simply check if `imageCapture` is null?